runhouse/servers/obj_store.py

```diff
@@ -1514,28 +1521,193 @@ def status_local(self):
                     else (py_module + "." + cls.__qualname__)
                 )
                 if isinstance(v, runhouse.Resource):
-                    resources_in_env_modified.append(
+                    objects_in_env_modified.append(
                         {"name": k, "resource_type": cls_name}
                     )
                 else:
-                    resources_in_env_modified.append(
+                    objects_in_env_modified.append(
                         {"name": k, "resource_type": cls_name}
                     )
-            return resources_in_env_modified
-        else:
-            return []
+
+        if env_name:
+            list_env_actor = (
+                subprocess.check_output(
+                    [
+                        "ray",
+                        "list",
+                        "actors",
+                        "-f",
+                        "state=ALIVE",
+                        "-f",
+                        f"name={env_name}",
+                    ]
+                )
+                .decode("utf-8")
+                .split("\n")
+            )
+
+            total_memory = psutil.virtual_memory().total
+
+            # The 'real' info about the env_servlet actors starts at index 9 of the list which represent the cli output.
```

Comment:
wondering if there's a more reliable way to parse this? e.g. maybe converting it to a dict rather than assuming the ordering of the info